auth4genai/intro/account-linking.mdx

```diff
@@ -7,50 +7,29 @@ description: "Account linking in Auth0 allows a user to connect multiple identit
-
-### Client-initiated account linking for AI agents
-
-**Client-initiated account linking** is a new option that provides a seamless way for users to grant AI agents access to different services. When an AI agent needs to access a new service (e.g., Google Calendar) for which it doesn't have a token, the application can trigger the account linking flow.
+### How it works
 
-<Frame>
-  ![Client-initiated account linking
-  flow](/img/client_initiated_account_linking.png)
-</Frame>
+Once a user successfully connects and authorizes access to a supported external provider, Auth0 adds: 
 
-The key steps are:
+- The account on the user profile, now known as a connected account.
+- The external provider’s access and refresh tokens for that connected account to the [Token Vault](/intro/token-vault).
 
-<Steps>
-  <Step title="Trigger the flow">
-    The client application initiates the flow when an agent needs access to a
-    new external provider.
-  </Step>
-  <Step title="User authentication">
-    The user is prompted to log in with the new account they wish to link.
-  </Step>
-  <Step title="Token exchange and linking">
-    Auth0 handles the authentication with the new provider and, upon successful
-    login, links the new identity to the user's primary profile.
-  </Step>
-  <Step title="Token retrieval">
-    With the accounts now linked, your application can use Token Vault to
-    retrieve an access token for the newly connected service to use for API
-    calls.
-  </Step>
-</Steps>
+Auth0 adds an account to the user profile using the [Connected Accounts flow](https://auth0.com/docs/secure/tokens/token-vault/connected-accounts-for-token-vault). When a user logs in via a supported external provider through Auth0, they initiate a Connected Accounts request from the client application: 
 
-This process ensures that the user maintains control and explicitly grants permission for each new service the agent needs to access, enhancing security and trust.
+1. The client application makes a `POST` request to the `/me/v1/connected-accounts` endpoint, passing scopes to send to the external provider.
+2. Auth0 creates a unique session ID and URL that redirects the user to a web browser. The client application saves the session ID for later verification.
+3. The client application redirects the user to URL, which opens in a browser, for user authentication and authorization with the Auth0.
```

Comment:
here it's not really "with the Auth0", right? I think this step is more about authenticating+authorizing with the external provider.

the application actually redirects to auth0 which will then immediately redirect to the external provider.